export/export_test.go

```diff
@@ -122,12 +130,26 @@ func TestZipMixedFilesAndFoldersAndExportOutput(t *testing.T) {
 }
 
 func requireEnvmanContainsValueForKey(t *testing.T, key, value, envmanStorePath string) {
-	b, err := ioutil.ReadFile(envmanStorePath)
+	b, err := os.ReadFile(envmanStorePath)
 	require.NoError(t, err)
-	envstoreContent := string(b)
 
-	t.Logf("envstoreContent: %s\n", envstoreContent)
-	require.Equal(t, true, strings.Contains(envstoreContent, "- "+key+": "+value), envstoreContent)
```

Comment:
This poor man's YAML parsing breaks when `value` is more complex and gets surrounded by quotes.